tsconfig.json

```diff
@@ -1,18 +1,26 @@
 {
     "compilerOptions": {
-        "target": "es2020",
-        "module": "es2020",
+        "target": "es2019",
```

Comment:
why?

Reply:
Node 12 needs es2019 as the target, according to https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-12